repository_creation/create_repository.go

```diff
@@ -9,26 +9,41 @@ import (
 
 	"github.com/ONSdigital/dp-cli/project_generation"
 	"github.com/ONSdigital/log.go/v2/log"
-	"github.com/google/go-github/v28/github"
+	"github.com/google/go-github/v66/github"
 	"github.com/spf13/cobra"
 	"golang.org/x/oauth2"
 )
 
 const (
-	org = "ONSdigital"
-	// Team: "DigitalPublishing", slug: "digitalpublishing", id: 779417
-	dpTeamSlug = "DigitalPublishing"
-	teamID     = int64(779417)
+	org        = "ONSdigital"
 	mainBranch = "main"
 )
 
+type GithubTeam struct {
+	slug string
+	name string
+}
+
+// List of standard Github Teams to apply permissions
```

Comment:
GoLand complains if there's not a capital H in GitHub

```suggestion
// List of standard GitHub Teams to apply permissions
```